v21.2/import-performance-best-practices.md

```diff
@@ -48,6 +48,10 @@ CockroachDB imports the files that you give it, and does not further split them.
 If you split the data into _more_ files than you have nodes, it will not have a large impact on performance.
 {{site.data.alerts.end}}
 
+### File storage during import
+
+During migration, all of the features of [`IMPORT`][import] that interact with external file storage assume that every node has the exact same view of that storage. In other words, in order to import from a file, every node needs to have the same access to that file.
```

Comment:
The link isn't rendering here for `IMPORT`. I think you want to adjust to parenthesis with import.html?